react/components/Table/totalizers/TotalizerIcon.js

```diff
@@ -0,0 +1,36 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+
+const CIRCLE_DIAMETER = '2.4rem'
+
+class TotalizerIcon extends PureComponent {
+  render() {
+    const {
+      item: { icon, iconBackgroundColor },
```

Comment:
There's no iconBackgroundColor in shape of prop.

Also I think it's better as it is in the shape, limiting the option of colors to success and error

Reply:
I've discussed it with @rsimoens and we will let it "open".